apps/playground-web/src/components/headless-ui/account-examples.tsx

```diff
@@ -0,0 +1,509 @@
+            <button
+              type="button"
+              className="flex min-w-[200px] flex-row items-center gap-2 rounded-md border bg-gray-800 px-4 py-2"
+            >
+              <AccountAvatar
+                className="h-10 w-10 rounded-full"
+                loadingComponent={
+                  <AccountBlobbie className="h-10 w-10 rounded-full" />
+                }
+                fallbackComponent={
+                  <AccountBlobbie className="h-10 w-10 rounded-full" />
+                }
+              />
+              <div className="flex flex-col gap-1">
+                <AccountName
+                  className="text-left"
+                  loadingComponent={
+                    <AccountAddress
+                      formatFn={shortenAddress}
+                      className="text-left"
+                    />
+                  }
+                  fallbackComponent={
+                    <AccountAddress
+                      formatFn={shortenAddress}
+                      className="text-left"
+                    />
+                  }
+                />
+                <div className="flex flex-row items-center gap-2">
+                  <AccountBalance
+                    className="text-left text-sm"
+                    chain={ethereum}
+                    loadingComponent={
+                      <span className="text-left text-sm">Loading...</span>
+                    }
+                    fallbackComponent={
+                      <span className="text-left text-sm">Loading...</span>
+                    }
+                  />
+                  <AccountBalance
+                    className="text-left text-sm"
+                    chain={ethereum}
+                    showBalanceInFiat="USD"
+                  />
+                </div>
+              </div>
+            </button>
+          </AccountProvider>
+        }
+        code={`import { AccountProvider, AccountAvatar, AccountName, AccountBalance, AccountAddress, AccountBlobbie } from "thirdweb/react";
+
+function App() {
+  return (
+    <AccountProvider address={vitalikAddress} client={THIRDWEB_CLIENT}>
+      <button
+        type="button"
+        className="flex min-w-[200px] flex-row items-center gap-2 rounded-md border bg-gray-800 px-4 py-2"
+      >
+        <AccountAvatar
+          className="h-10 w-10 rounded-full"
+          loadingComponent={
+            <AccountBlobbie className="h-10 w-10 rounded-full" />
+          }
+          fallbackComponent={
+            <AccountBlobbie className="h-10 w-10 rounded-full" />
+          }
+        />
+        <div className="flex flex-col gap-1">
+          <AccountName
+            className="text-left"
+            loadingComponent={
+              <AccountAddress
+                formatFn={shortenAddress}
+                className="text-left"
+              />
+            }
+            fallbackComponent={
+              <AccountAddress
```

Comment:
wonder if this should be the fallback by default. feels like most ppl would want that

Reply:
Does making it return a different component, makes it impure? Personally I'm happy with making things a bit redundant (or verbose, if you will) in return for what we call a true headless component